server/test/handlers/updateAvailability.test.js

```diff
@@ -0,0 +1,8 @@
+const workerTimeslotHandler = require('../../models/handlers/workerTimeslot');
+
+describe('testing inputting of worker schedule functionality', () => {
+    beforeEach(() => {
+        jest.resetModules(); // Clears any cache between tests.
+    });
+ 
+});
```

Comment:
Hi Simran,

This is a version 1 of the PR, so I understand there are no tests created yet. However, to proceed with this PR, I'd recommend writing automated test cases to cover the `updatingTheWorkerAvailability` method.

Reply:
Hi Nancy! 

Thanks a lot! Peter and I have created 3 tests in our latest PR to resolve this comment. Thank you for your time! 

Simran

Reply:
Hi Simran,

I took a look at your test cases, thank you for adding those in!
I noticed that for test case 1 and 2, the status of the workerTimeslotId = 5 and 7 must be unavailable. The same database is being used by multiple methods which edit the status of the entries in the worker_Timeslot table (eg. S19-T2 changes the status from unavailable to available when an appointment is cancelled). Due to this, the value of the status parameter is constantly changing. Since your tests return true (indicating that the slot is unavailable), these tests may not return true if the value of status in the database for workerTimeslotId = 5 and 7 is unavailable.

Reply:
Hi Nancy, 

Thank you very much for bringing this point up. Peter and I had to carefully choose workerTimeslotId 5 and 7 to test since it's not being changed or altered by other tests. In this case the status will not change from available to unavailable by another test other than ours. 

Thank you very much for bringing such a critical point up! I appreciate it! 

Simran